vendor/github.com/magiconair/properties/doc.go

```diff
@@ -102,7 +102,7 @@
 //   v = p.GetString("key", "def")
 //   v = p.GetDuration("key", 999)
 //
-// As an alterantive properties may be applied with the standard
+// As an alternative properties may be applied with the standard
```

Comment:
Please do not modify vendor/ directory - those files are managed by godeps and we are not modifying them manually.